### PR TITLE
Prepare for 0.12 Release

### PR DESCRIPTION
**Pull request by nolim1t**
Prepare for 0.12 Release

### CODE REVIEW

**Comment by nolim1t** [comment]
As this stands, this project will not build due to a mismatch in **@roasbeef** keys. It has been pointed out in the [lnd issues](https://github.com/lightningnetwork/lnd/issues/4970)

**Comment by Roasbeef** [comment]
> As this stands, this project will not build due to a mismatch in @Roasbeef keys. 

So your system parses the release notes to figure out what key to verify against?

File: 0.12/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+ARG VERSION
+
+RUN apk add --no-cache  gnupg  git
+
+ENV KEYBASE_USER1=roasbeef
+ENV KEYBASE_USER2=bitconner
+
+# roasbeef and conner keys
+ENV KEYS 4AB7F8DA6FAEBB3B70B1F903BC13F65E2DC84465 9C8D61868A7C492003B2744EE7D737B67FA592C7
```
Comment (Roasbeef) [inline 0.12/Dockerfile:43]: Ah ok, this should be `E4D85299674B2D31FAA1892E372CBD7633C61696`

Comment (nolim1t) [reply 0.12/Dockerfile:43]: Thanks

**Pull request merged**